Makefile

```diff
@@ -1,48 +1,50 @@
 .PHONY: dev lint complex coverage pre-commit yapf sort deploy destroy deps unit infra-tests integration e2e coverage-tests docs lint-docs build
+CWD := $(shell pwd)
 
-
-
+.ONESHELL:  # run all commands in a single shell, ensuring it runs within a local virtual env
 dev:
+	python3 -m venv .venv
```

Comment:
what about the windows makefile variation? they are now different

Reply:
Both synchronized now.